api/.env.rebuild_staging

```diff
@@ -14,4 +14,5 @@ PUSH_NOTIFICATION_BACKEND=pcapi.notifications.push.backends.logger.LoggerBackend
 SEARCH_BACKEND=pcapi.core.search.backends.testing.TestingBackend
 SIRENE_BACKEND=pcapi.connectors.entreprise.backends.testing.TestingBackend
 SMS_NOTIFICATION_BACKEND=pcapi.notifications.sms.backends.logger.LoggerBackend
+SUBCATEGORY_SUGGESTION_BACKEND=pcapi.core.external.subcategory_suggestion_backends.subcategory_suggestion.SubcategorySuggestionBackend
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
**💣 chore:** ‏pas besoin de ça ici, elle fait doublon avec la valeur dans .env.staging
.env.rebuild_staging est chargé _en plus_ des variables d'env (de staging du coup)